docs/data/sdks/index.md

```diff
@@ -438,38 +445,42 @@ description: The quick start guide of how to use amplitude SDK
 
 ## 3. Check for success
 
+After you begin sending data to Amplitude, use one of these tools to check your instrumentation and validate your events.
+
 ### Ingestion Debugger
 
-1. Login to you Amplitude Analytics Orgnization.
-2. Click the `Data Source tab` at the bottom of the navigation bar
-3. Choose the `Ingestion Debugger` section. You can check the Requests or which User or Device has been throttled.
+Use the Ingestion Debugger in Amplitude to check your requests, event and identify counts, and throttled users or devices.
+
+1. Log in to Amplitude.
+2. Click **Data Source** at the bottom of the navigation bar.
+3. Click the `Ingestion Debugger` tab. 
```

Comment:
Should we also use the strong format for `Ingestion Debugger`?
Like **Ingestion Debugger**.

Reply:
Good catch, I'll fix it before I merge!